WordPressCom-Stats-iOS/StatsTableViewController.m

```diff
@@ -1345,5 +1349,27 @@ - (void)resetDateToTodayForSite
     self.selectedDate = date;
 }
 
+- (void)trackViewControllerAnalytics
+{
+    WPAnalyticsStat stat;
+    
+    switch (self.selectedPeriodUnit) {
+        case StatsPeriodUnitDay:
+            stat = WPAnalyticsStatStatsPeriodDaysAccessed;
+            break;
+        case StatsPeriodUnitWeek:
+            stat = WPAnalyticsStatStatsPeriodWeeksAccessed;
+            break;
+        case StatsPeriodUnitMonth:
+            stat = WPAnalyticsStatStatsPeriodMonthsAccessed;
+            break;
+        case StatsPeriodUnitYear:
+            stat = WPAnalyticsStatStatsPeriodYearsAccessed;
+            break;
+    }
+    
+    [WPAnalytics track:stat];
```

Comment:
Would it make sense to have a `StatsPeriodUnitUnknown`?  I ask just to cover all cases but it looks like `self.selectedPeriodUnit` should never be `nil` so maybe its not needed.